### ISSUE TITLE
Is there any way to make the website stream interactive 

### ISSUE THREAD

**Issue by Tharusha-dev** [jeffshee/hidamari#124]
For example like in [this](https://alteredqualia.com/xg/examples/shadow_profiles.html) webpage, you can move the mouse to interact with the image.

**Comment by Elmapul** [comment]
interaction with keyboard dont work either

**Comment by Elmapul** [comment]
i figure out an way to interact with the an webpage , my solution will only be usefull for developers who are making their own sites, and it cant interact with the mouse buttons, its an massive "work arround", but let me know if you are interessed

**Comment by Tharusha-dev** [comment]
Absolutely!  I'm sure there are open source websites that's similar to what I linked earlier, so maybe your method can be used with them.

**Comment by Elmapul** [comment]
hold a few minutes

**Comment by Elmapul** [comment]
here it is:
https://github.com/Elmapul/interactive-wallpaper-template-for-hidamari/tree/main

i tried my best to write install/use instructions, but it might be a bit confusing.
the code might be a bit confusing to read as well, let me know if you have any question (just open an issue on my repository)
